autogen/main/cluster.tf.tmpl

```diff
@@ -111,6 +111,7 @@ resource "google_container_cluster" "primary" {
         {% if beta_cluster %}
         min_cpu_platform = lookup(var.node_pools[0], "min_cpu_platform", "")
         {% endif %}
+        image_type = lookup(var.node_pools[0], "image_type", "")
```

Comment:
Looks like there is a validation that prevents empty string as default. Can we try null instead?
https://github.com/hashicorp/terraform-provider-google/blob/627220d3985030c1a65c87faa9ea882e46dc0e63/google/resource_container_cluster.go#L341

Reply:
null is not accepted either, trying with the default value "COS_CONTAINERD"

Reply:
@tweiss-mdm thats reasonable